packages/rolldown/src/cli/load-config.ts

```diff
@@ -52,16 +52,17 @@ async function bundleTsConfig(configFile: string): Promise<string> {
       },
     ],
   })
+  const outputDir = path.dirname(configFile)
   const result = await bundle.write({
-    dir: path.dirname(configFile),
+    dir: outputDir,
```

Comment:
Is using `file` more proper in this case? @underfin cc

Reply:
Here need to generate a hash filename, the `file` option is not suitable.

Reply:
Does `file` not support `[hash]` placeholder?